### PR TITLE
Hw6 get todos from server

### PR DESCRIPTION
**Pull request by Armensch11**
Screen Shot 2023-04-14 at 2 08 01 PM
Screen Shot 2023-04-14 at 2 08 23 PM
Screen Shot 2023-04-14 at 2 09 10 PM
Screen Shot 2023-04-14 at 2 25 04 AM
Screen Shot 2023-04-14 at 2 25 11 AM

### CODE REVIEW

File: HW5/functional/modalActions.js [hunk 1]
```diff
@@ -99,21 +85,33 @@ export function validateEntries() {
...
   console.log(todoItem);
 }
-export function addTask(todoItem) {
-  const existingTodos = JSON.parse(localStorage.getItem("todos")) || [];
-  existingTodos.push(todoItem);
-  localStorage.setItem("todos", JSON.stringify(existingTodos));
-  // console.log(existingTodos);
+async function addTask(todoItem) {
+  try {
+    const saveToRemote = await fetch("http://localhost:3005/tasks", {
```
Comment (younaAnn) [inline HW5/functional/modalActions.js:97]: You can just call await fetch without variable

File: HW5/functional/modalActions.js [hunk 2]
```diff
@@ -89,7 +75,7 @@ export function validateEntries() {
   )[0];
 
   if (todo && dueDate && todoType) {
-    console.log("validating ");
+    // console.log("validating ");
     todoItem.task = todo;
     todoItem.date = dueDate;
     todoItem.type = todoType;
```
Comment (younaAnn) [inline HW5/functional/modalActions.js:81]: You cannot make
todoItem.id = Math.floor(Math.random() * 10000).toString();
If you use this id later (and you send it to server, so you use)
id is the UNIQUE id, but Math.random() don't guarantee that. You can generate id using timestamp UTC, for example

File: HW5/server/db.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    {
+      "task": "go runnin",
+      "date": "Thursday, 13  April",
+      "type": "health",
+      "status": "pending",
+      "id": "7597"
+    },
+    {
+      "task": "do the insert of string into html",
+      "date": "Today",
```
Comment (younaAnn) [inline HW5/server/db.json:19]: You shouldn't store dates like "Today" in database - it's just impossible to know which date the "Today" was :)

Comment (Armensch11) [reply HW5/server/db.json:19]: yes it was a mistake to store that , it should have been evaluated to "today" or "tommorow " during tasks render

**Pull request merged**